### DISCUSSION TITLE
大家平常都在哪裡看新聞？

### DISCUSSION THREAD

**Discussion by vchien03** [1101-GS4719-C/Python-Community-GS4719-C#1874] [General]
小時候住家裡，電視幾乎整天開著播新聞或討論節目，就算沒特別關心，也會無意間聽到時事。但現在住校，我平常也不會特別關注新聞，感覺快要跟社會脫節了。
目前我知道的渠道有line today、手機版google (但會被大數據影響，都變成我有興趣的內容)，想問大家平時都用甚麼方式關注新聞，或是還有哪種管道了解時事嗎？

**Comment by TobywithDino** [comment]
長大之後平日都待在自己的房間中，也很少跟家人看電視，自然而然就很少在看新聞了
但我發現在中央，有許多早餐店都有電視，因為在吃早餐不知道要幹嘛就會看新聞，這時也許是我唯一和社會接軌的時候了吧...

**Comment by tzurou** [comment]
我自己也是用line today看新聞比較多，因為平常用賴無聊的時候就順手看看有沒有甚麼大新聞，我也有下載天下雜誌的App，主要可以看到比較多有深度的國際新聞，但我沒有訂閱所以都只能看幾篇免費版的新聞，如果想要主動接收更多新聞的話可以訂閱天下雜誌，應該會蠻有收穫的。

**Comment by wanka2030** [comment]
小時候都會跟著家人一起收看新聞，無論是晚餐時間或早上要出門之前會看看氣象預報，但長大後有可能是因為網路的普及又或者是我們早已成為了網路的一份子，隨時隨地會從別人的口中、手機訊息的推波來接收外界的新聞訊息:D
但還是會懷念新聞充滿補充或是節外生枝的問題，都會在吸收資訊之後加以思考問題的發生。

**Comment by thomasxshen** [comment]
待在家裏，平常閒閒沒事幹就會關心一下時事，或者看一些時事評論，為了今後從政做準備。

**Comment by LinChloe** [comment]
平常我都不看新聞，偶爾滑滑Dcard，就會有近期最火爆的新聞在上面。
或著去早餐店吃早餐時，順便看下電視享受美好的早成時光。
但我覺得如果要看跟國際接軌的話多看點國外新聞頻道會比較好。

**Comment by happy246810** [comment]
我自己是下載新聞的app，這樣我就可以知道學校外發生了什麼事。除了新聞app，我也會在fb，google上看到新聞，只是這些新聞會依據我的習慣而被限制主題。我下載的app名字叫新聞雲，如果想要，也可以下載看看喔。

**Comment by YifenLin** [comment]
之前住在家裡，吃飯的時候都會配新聞。但現在出來住宿舍真的就不會點新聞來看，有一陣子跟朋友聊天發現自己跟社會大脫節，現在我接收新聞的管道第一個是instagram, 另一個是decard畢竟每天都會點開來看，大家討論甚麼就會看到了。

**Comment by SusuTima** [comment]
有時候會看Chrome主頁的新聞，也會看fb推薦的新聞，真的很有空會去翻商業周刊，看看最近的時事。

**Comment by jylin34** [comment]
主要看什麼類型的新聞，政治或時事會透過youtube或google 新聞看台灣的媒體，經濟方面我覺得bloomberg跟金十數據這兩個app不錯，會實時跳通知顯示現在有發生哪些事情，有些平台沒有app的，也可以透過訂閱rss 新文章自動發通知，我覺得很方便。

**Comment by willychiuchiu** [comment]
平常不太會特別去看新聞，但是平時在滑社群軟體時（IG、臉書、PTT等）看到別人分享，或是在YouTube 有推薦時就會稍微點進去看看最近發生什麼事

**Comment by Hahahahbla** [comment]
平常不太會特別去看新聞，但透過Line Today、Google、天下雜誌App等網路平台，社交媒體Dcard、Instagram等可以補足對時事的關注，可以避免與社會脫節。

**Comment by YuanWang08** [comment]
現在已經沒什在看新聞了，真正很嚴重的大事自然會傳到耳裡，因為現在當今台灣普遍媒體都在寫農場文，毫無價值可言，況且有些還資訊不實，記者捏造流量，素質不是比小學生差，而是想要提高收視度就必須下那種浮誇不實的標題，我個人是會訂閱youtube的bbc 與 cnn偶爾看看國際上發生了什麼事，比看台灣藝人在偷吃誰好多了。

**Comment by 20050201Shawn** [comment]
我現在都看一個youtube頻道叫異色檔案，很喜歡主持人的口條跟敘事方式，雖然他的影片比較像是社會議題而不是即時新聞，但這是我目前唯一關心社會議題的管道。

**Comment by yyujimmy** [comment]
新聞網站和應用程式：許多人會通過各種新聞網站和應用程式來獲取新聞，例如BBC News、CNN、The New York Times、The Guardian等等。這些網站和應用程式提供了即時的新聞更新和各種不同類型的報導。
社交媒體：許多人也會透過社交媒體平台獲取新聞，例如Twitter、Facebook、LinkedIn等。這些平台通常會有許多媒體組織的官方帳號，可以直接在上面瀏覽新聞內容。
電視新聞：電視新聞節目仍然是許多人獲取新聞的主要來源之一。人們可以通過觀看國內和國際的新聞頻道，如CNN、BBC、FOX News等來了解最新的消息。
報紙和雜誌：雖然逐漸被數字化取代，但仍有人喜歡閱讀傳統的印刷報紙和雜誌。這些媒體提供了深度報導和分析，有助於人們更全面地了解新聞事件。
Podcast和網絡節目：一些人喜歡通過聽播客或網絡節目的方式來獲取新聞和分析。這些節目通常由專家或評論員主持，提供了更加深入和詳盡的報導。

**Comment by KH1126** [comment]
現在都是通過手機來觀看新聞。
像是google新聞或者是有時候用CNN看國際的新聞來學習英文。
但最近比較多是使用Fb來關注現在的新聞。

**Comment by William08290829** [comment]
以前小時候電視兒童都是跟爸媽看新聞看來的，但現在家裡根本沒人在看電視只能從手機的推薦新聞偶而看看。但老實說我覺得只從這樣的管道了解新聞真的偏少，基本上只會知道那種超大條的新聞。

**Comment by Eileen050824** [comment]
平常在學校大多在忙課業的事，老實說沒什麼在follow新聞，頂多等餐的時候看看google給我推薦的內容，如同上面所說的確比較符合我的喜好(畢竟有大數據過濾)。等到寒暑假回家，陪著家人看新聞的時候，頓時有種原來我miss那麼多的感覺，更會關心目前社會情勢的狀況(有時甚至有點杞人憂天XD)！

**Comment by YOGA012** [comment]
我現在都是用LINE看新聞，感覺很方便。LINE Today會自動推送各種最新的新聞和時事，內容也很豐富，涵蓋了國內外新聞、娛樂、科技等各個方面。雖然有時候也會被大數據影響推送的內容，但整體上還是能掌握到不少重要的資訊。如果你也有用LINE，這或許是一個不錯的選擇來關注新聞。

**Comment by woohsin** [comment]
平常不看新聞

**Comment by alexender0104** [comment]
讀大學前住家裡，平時都是吃飯時和家人一起看新聞配飯吃，常常一個小時的新聞，不到30分鐘就在重播重複的內容，上了大學後都是用line看新聞，會自動推送我平常有在關注的議題，以及國內外的重大事件，我覺得挺方便的，比起以前看新聞台，只能被動接受，現在能自己找有興趣的內容看挺好的。

**Comment by hbh10601** [comment]
讀大學之後真的很少接觸，平常新聞來源可能都是Youtube上推薦來的，不然就是偶爾會看到FB、ig瘋狂轉傳的大事

**Comment by cassy0201** [comment]
以前都是在家裡跟著爸媽看，長大後偶爾想到會曾google看，或是滑臉書也會看到一些新聞片段，但兩者都會被演算法影響，容易出現自己平常有關注的內容，而不是普通新聞

**Comment by NcuJerryKao** [comment]
了解時事是非常重要的，可以幫助我們擴展視野、增進知識，並且與社會保持連結。除了line today和手機版google，還有許多其他管道可以獲取新聞資訊，以下是一些建議：
1. 新聞網站與應用程式：許多新聞機構都有自己的網站和應用程式，如蘋果日報、自由時報等，可以透過他們的平台了解最新的新聞資訊。
2. 社群媒體：許多社群媒體平台上也有新聞資訊，如臉書、推特等，可以追蹤新聞機構或相關帳號，獲取最新消息。
3. 訂閱電子報：許多新聞網站提供電子報訂閱服務，可以每天收到最新的新聞摘要。
4. 收聽廣播或播客：有些廣播節目或播客專門討論時事議題，可以透過聽覺方式了解新聞。
5. 閱讀報紙：雖然現在網路新聞發達，但有時候閱讀傳統報紙也是一種不錯的方式來了解新聞。
希望這些建議對您有所幫助，讓您能夠更方便地關注時事，保持對社會的關注與連結。

**Comment by yubo0oo** [comment]
公共電視台
